node/src/components/contract_runtime/operations.rs

```diff
@@ -99,3 +199,52 @@ pub(super) async fn execute(
     trace!(?result, "execute result");
     result
 }
+
+fn commit_step(
+    engine_state: &EngineState<LmdbGlobalState>,
+    metrics: &ContractRuntimeMetrics,
+    protocol_version: ProtocolVersion,
+    pre_state_root_hash: Digest,
+    era_report: &EraReport<PublicKey>,
+    era_end_timestamp_millis: u64,
+    next_era_id: EraId,
+) -> Result<StepSuccess, StepError> {
+    // Extract the rewards and the inactive validators if this is a switch block
+    let EraReport {
+        // Note: Highway does not slash, do nothing with the equivocators
```

Comment:
Should we log an error if it's not empty? Or just evict them? (That's what https://github.com/casper-network/casper-node/pull/1637 will do anyway.)

Reply:
I went ahead and evicted them in c4757fec6839f056de203cd5f9da8a8dffa1e29f